.gitignore

```diff
@@ -13,7 +14,16 @@ logs
 /jsettlers.common/resources/images/*
 !/jsettlers.common/resources/images/movables.txt
 /jsettlers.common/resources/save
+config.prp
 
 /jsettlers.main.swing/*.log
 
-/jsettlers.tests/out
+#IDE Files
```

Comment:
IDE specific files (except eclipse because it's the official build platform) should be handled in `.git/info/exclude` because the number of IDEs and editor is endless and thus `.gitignore` might have an endless length one day.